public/js/event-app.js

```diff
@@ -120,25 +120,34 @@ $(document).ready(function() {
         var firstScriptTag = document.getElementsByTagName('script')[0];
         firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
+        this.currentUser = new models.User(USER);
+        this.currentUserIsAdmin = function() {
+          return this.currentUser.isAdminOf(curEvent);
```

Comment:
I believe you that it works.  I'm concerned about the code style of going 3 levels deep in the call stack with ``this`` and having it work -- it seems brittle; any variation in how those methods get invoked and the binding is lost.

We could either explicitly bind "this" in the function definitions (e.g. ``(function() {}).bind(this)``), or enclose a reference to this which isn't dynamic, e.g.:

    var that = this
    that.currentUserIsAdmin = function() {
        return that.currentUser.isAdminOf(curEvent);
    };

Just concerned about maintainability of that code.